src/AppBundle/Controller/FrontController.php

```diff
@@ -18,11 +22,15 @@ class FrontController extends Controller
     public function indexAction(Request $request)
     {
         // replace this example code with whatever you need
+        $parser = $this->get(Parser::class);
+        $parser->feed("https://kalaxia.org/?feed=rss2");
+
         return $this->render('front/index.html.twig', [
             'base_dir' => realpath($this->getParameter('kernel.project_dir')).DIRECTORY_SEPARATOR,
+            'tickets' => $parser->items,
```

Comment:
You could avoid using ``$parser`` variable by putting the whole thing in one line:

```
'tickets' => $this->get(Parser::class)->feed("https://kalaxia.org/?feed=rss2")
```

``feed()`` should return the parsed items, rather than setting a public property used later.